examples/plot_knockoffs_wisconsin.py

```diff
@@ -0,0 +1,237 @@
+from sklearn.preprocessing import StandardScaler
+
+data = load_breast_cancer()
+X = data.data
+y = data.target
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.1, random_state=seed
+)
+
+scaler = StandardScaler()
+X_train = scaler.fit_transform(X_train)
+X_test = scaler.transform(X_test)
+n_train, p = X_train.shape
+n_test = X_test.shape[0]
+feature_names = [str(name) for name in data.feature_names]
+
+
+#############################################################################
+# Selecting variables with the Lasso
+# ------------------------------------------
+# We want to select variables that are relevant to the outcome, i.e. tumor
+# charateristics that are associated with tumor malignance. We start off by applying a
+# classical method using Lasso logistic regression and retaining variables with non-null
+# coefficients:
+from sklearn.linear_model import LogisticRegressionCV
+
+clf = LogisticRegressionCV(
+    Cs=np.logspace(-3, 3, 10), penalty="l1", solver="liblinear", random_state=rng
+)
+clf.fit(X_train, y_train)
+print(f"Accuracy of Lasso on test set: {clf.score(X_test, y_test):.3f}")
+
+
+selected_lasso = np.where(np.abs(clf.coef_[0]) > 1e-6)[0]
+print(f"The Lasso selects {len(selected_lasso)} variables:")
+print(f"{'Variable name':<30} | {'Coefficient':>10}")
+print("-" * 45)
+for i in selected_lasso:
+    print(f"{feature_names[i]:<30} | {clf.coef_[0][i]:>10.3f}")
+
+
+#############################################################################
+# Evaluating the rejection set
+# ------------------------------------------
+# Since we do not have the ground truth for the selected variables (i.e. we do not know
+# the relationship between the tumor characteristics and the the malignance of the tumor
+# ), we cannot evaluate this selection set directly. To investigate the reliability of
+# this method, we artificially increase the number of variables by adding noisy copies
+# of the features. These are correlated with the variables in the dataset, but are not
+# related to the outcome.
+repeats_noise = 5  # Number of synthetic noisy sets to add
+
+noises_train = [X_train]
+noises_test = [X_test]
+feature_names_noise = [x for x in feature_names]
+for k in range(repeats_noise):
+    X_train_c = X_train.copy()
+    X_test_c = X_test.copy()
+    noises_train.append(X_train_c + 2 * rng.randn(n_train, p))
+    noises_test.append(X_test_c + 2 * rng.randn(n_test, p))
+    feature_names_noise += [f"spurious #{k*p+i}" for i in range(p)]
+
+noisy_train = np.concatenate(noises_train, axis=1)
+noisy_test = np.concatenate(noises_test, axis=1)
+
+
+#################################################################################
+# There are 180 features, 30 of them are real and 150 of them are fake and independent
+# of the outcome. We now apply the Lasso (with cross-validation to select the best
+# regularization parameter) to the noisy dataset and observe the results:
+lasso_noisy = LogisticRegressionCV(
```

Comment:
The number of jobs should be defined for this class.

Reply:
I removed `n_jobs`, it's not critical in this light example.

Reply:
You should put it at 1.
I want to have control over the different n_jobs in the examples.